### DISCUSSION TITLE
Upgrading react-admin to v3.0.0 or higher

### DISCUSSION THREAD

**Discussion by mathiasvanimpe** [mozilla/hubs#3672] [Hubs Cloud Questions] [answered]
Hey Team,

First of all a Happy New Year!

I just wanted to see if upgrading the `react-admin` package to version v3.0.0 or higher is anywhere on the roadmap for you guys. The reason why I'm asking is because currently a dependency of `react-admin` called `papaparse` is flagged as a high severity vulnerability. (react-admin@2.9.6 requires papaparse@^4.1.4 via ra-ui-materialui@2.9.6). Versions of `papaparse` prior to 5.2.0 are vulnerable to Regular Expression Denial of Service (ReDos).

Bumping `react-admin` should resolve the issue since they [removed the papaparse package ](https://github.com/marmelab/react-admin/blob/93e7bb68bc9d827738a7fd5f75f9d1d90c0cb115/CHANGELOG.md#v300-alpha0) in v3.0.0-alpha0

I tried to upgrade `react-admin` myself but ran into some issues with the postgres-data-provider and could not resolve it since I could not run the admin panel locally to properly debug. 

Cheers!

**Comment by robinkwilson** [comment, marked as answer]
Created an issue in hubs-cloud repo, follow it here: https://github.com/mozilla/hubs-cloud/issues/155
I'll see what we can do. Thank you for flagging!

**Reply by mathiasvanimpe** [reply]
Great. Thanks Robin!
